bot/exts/christmas/advent_of_code/_helpers.py

```diff
@@ -48,7 +48,7 @@
 )
 
 # Create an easy constant for the EST timezone
-EST = pytz.timezone("EST")
+EST = "America/Chicago"
```

Comment:
According to @janine9vn 
> Chicago is CST/CDT. So one hour before EST/EDT

[New York](https://www.zeitverschiebung.net/en/timezone/america--new_york), should be the right timezone.